mlir/test/fusion/e2e/resnet50/mixr-resnet-fusion-case-1.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: rocmlir-opt -migraphx-to-tosa %s | rocmlir-driver -host-pipeline partition,highlevel -targets %arch | rocmlir-gen -ph -print-results -rand 4 -rand_type float -fut test --verifier clone - | rocmlir-driver -host-pipeline xmodel,runner -kernel-pipeline full -targets %arch | mlir-cpu-runner -O2 --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CLONE
```

Comment:
I suppose we don't need `-host-pipeline partition` here because currently we rely on migraphx partitioning.

Reply:
Im simply being consistent with other mixr clone tests.  

https://github.com/ROCmSoftwarePlatform/rocMLIR/blob/ca6015d47d9f73ace4c2eda3fc9059dfab27bdd4/mlir/test/fusion/e2e/mixr-gemm/mixr-gemm-add.e2e.mlir#L2

It also fails w/o partition maybe because we use clone verification as follows : 

 'RUN: at line 1';   /home/mkarunar/work/rocMLIR/build/bin/rocmlir-opt -migraphx-to-tosa /home/mkarunar/work/rocMLIR/mlir/test/fusion/e2e/resnet50/mixr-resnet-fusion-case-1.mlir | rocmlir-driver -host-pipeline highlevel -targets gfx908:sramecc+:xnack- | rocmlir-gen -ph -print-results -rand 4 -rand_type float -fut test --verifier clone - | rocmlir-driver -host-pipeline xmodel,runner -kernel-pipeline full -targets gfx908:sramecc+:xnack- | mlir-cpu-runner -O2 --shared-libs=/home/mkarunar/work/rocMLIR/build/external/llvm-project/llvm/lib/libmlir_rocm_runtime.so,/home/mkarunar/work/rocMLIR/build/lib/libconv-validation-wrappers.so,/home/mkarunar/work/rocMLIR/build/external/llvm-project/llvm/lib/libmlir_runner_utils.so,/home/mkarunar/work/rocMLIR/build/external/llvm-project/llvm/lib/libmlir_c_runner_utils.so --entry-point-result=void | /home/mkarunar/work/rocMLIR/build/external/llvm-project/llvm/bin/FileCheck /home/mkarunar/work/rocMLIR/mlir/test/fusion/e2e/resnet50/mixr-resnet-fusion-case-1.mlir --check-prefix=CLONE
--
Exit Code: 2

Command Output (stderr):
--
Warning: unknown architecture, falling back to defaults: 
error: failed to legalize operation 'builtin.unrealized_conversion_cast' that was explicitly marked illegal

Reply:
PS I didnt mean to highlight the error its just copy-pasting error and markdown being overly smart here :)

Reply:
I wonder if there is any way to check if partitioning pass actually split the given func.
@sjw36 can it partition a func? can it run a module when it has more than one func in it?

Reply:
Partitioning only applies to non-kernel funcs and yes it will work for multiple funcs.